react/components/EXPERIMENTAL_Table/typings.d.ts

```diff
@@ -0,0 +1,16 @@
+interface Column {
+  title: string
```

Comment:
With this are we going to remove the support to internationalisation components as title entirely?

Reply:
You could use `formatMessage` function for that. But I can add the support for the `<FormattedMessage />` component,  though.

Reply:
This is an issue we were discussing actually. It would be nice to accept formatted message, but some people don't want to risk accepting "anything". It might be worth discussing with more people.